src/app/space/create/codebases/services/codebases.service.ts

```diff
@@ -110,6 +112,24 @@ export class CodebasesService {
       });
   }
 
+  /**
+   * Delete codebase
+   *
+   * @param {Codebase} codebase codebase to delete
+   * @returns {Observable<Codebase>}
+   */
+  delete(codebase: Codebase): Observable<Codebase> {
```

Comment:
`delete` is a reserved word in Typescript and so should not be used alone. Can you can it to something like `deleteCodebase`. 

Can you please update the unit test for this function.